external/fv3fit/tests/data_/test_windowed_zarr.py

```diff
@@ -63,13 +63,13 @@ def test_loader_stacks_default_config():
         )
         dataset = loader.open_tfdataset(
             local_download_path=None, variable_names=variable_names
-        )
+        ).unbatch()
         item = next(iter(dataset))
-        assert item["a"].shape[0] == NX * NY
-        assert len(item["a"].shape) == 3
-        assert item["a"].shape[-1] == NZ
-        assert item["a_sfc"].shape[0] == NX * NY
-        assert len(item["a_sfc"].shape) == 2
+        assert item["a"].shape[0] == 10
```

Comment:
Some commentary here  about what we're checking would be helpful, given that the test name alone doesn't indicate all of this. A related way of saying this is that it's hard to tell that this explicitly checks the PR's functionality of taking a sample from a stacked dimension. Can you add either comments/renaming or another test make it more explicit?

Reply:
Will do.

Reply:
Updated with a comment. These assertions are part of testing that the verb "stacks" in the test name was done right.